### DISCUSSION TITLE
Cache save fails - Cache service responded with 500

### DISCUSSION THREAD

**Discussion by max-nextcloud** [community/community#64484] [Actions] [answered]
### Select Topic Area

Product Feedback

### Body

Lately we've been seeing cache failures in our cypress jobs like this one:
https://github.com/nextcloud/text/actions/runs/5922556743/job/16061659723

Warning: Failed to save: reserveCache failed: Cache service responded with 500

I think this is running on github infrastructure and something is wrong with the cache service there. Please let me know where to report this if this is not the appropriate place.

**Comment by KocsisV** [comment]
I'm experiencing the same issue.

**Comment by nirav-gajera** [comment]
many peoples facing same issue you should contact [Github support](https://github.com/contact)

**Comment by ruiguerrapsyomics** [comment]
We are also experiencing this issue.

**Reply by shay-legit** [reply]
same 500 on cache write / read

**Comment by alisabrigok** [comment]
We are experiencing the very same issue. The error was `Warning: uploadChunk failed: Cache service responded with 500`, and now it's `Warning: Failed to save: reserveCache failed: Cache service responded with 500`. GitHub Status doesn't show any anomaly in their services, but something is clearly not working properly from their side.

**Comment by robinmacharg** [comment]
Ditto.  Here's my output.  This is for a clean setup with no cache previously created.  It doesn't fail outright, and reports success but no cache is seen on next run, or in the management UI.

```sh
Post job cleanup.
/usr/local/bin/gtar --posix -cf cache.tzst --exclude cache.tzst -P -C /Users/runner/work/REDACTED/PATH --files-from manifest.txt --delay-directory-restore --use-compress-program zstdmt
Warning: Failed to save: uploadChunk (start: 0, end: 33554431) failed: Cache service responded with 500
Cache saved with key: macOS-spm-40ac54985028e101eb810bb9a7d1e25fb9375c73c6dd_REDACTED
```

It's for an iOS app, trying to cache Swift Package Manager packages.  The action step to create/restore it prior to our `xcodebuild` command is uncontroversial, I think:

```yaml
- name: Cache configuration
   uses: actions/cache@v3
   with:
     path: ./SourcePackages
     key: ${{ runner.os }}-spm-${{ hashFiles('**/Package.resolved') }}
     restore-keys: |
       ${{ runner.os }}-spm-
```

**Comment by ralvarez-globant** [comment]
Same here, using cache@v2 (upgrading to v3 to see if anything changes).
`Warning: uploadChunk (start: 167772160, end: 201326591) failed: Cache service responded with 500`

**Comment by max-nextcloud** [comment, marked as answer]
Working again for me now. I'll keep this open for a day or two to see if people are still observing this.
